requirements/edx/base.in

```diff
@@ -78,7 +78,8 @@ edx-celeryutils
 edx-completion
 edx-django-release-util             # Release utils for the edx release pipeline
 edx-django-sites-extensions
-edx-django-utils
+# TODO(jinder): uncomment below after plugin code has been moved
+# edx-django-utils
```

Comment:
@ reviewer: I plan on uncommenting this once this has merged: https://github.com/edx/edx-django-utils/pull/48